tests/system-broker/tests/auth_test.go

```diff
@@ -95,6 +115,66 @@ func TestSystemBrokerAuthentication(t *testing.T) {
 	})
 }
 
+func TestCallingORDServiceWithCert(t *testing.T) {
+	ORDContext, err := pkg.NewTestContext(config)
+	require.NoError(t, err)
+
+	logrus.Infof("registering runtime with name: %s, within tenant: %s", runtimeInput.Name, ORDContext.Tenant)
+	runtime := director.RegisterRuntimeFromInputWithinTenant(t, ORDContext.Context, ORDContext.DexGraphqlClient, ORDContext.Tenant, runtimeInput)
+	defer director.UnregisterRuntimeWithinTenant(t, ORDContext.Context, ORDContext.DexGraphqlClient, ORDContext.Tenant, runtime.ID)
+
+	app, err := director.RegisterApplicationWithinTenant(t, ORDContext.Context, ORDContext.DexGraphqlClient, ORDContext.Tenant, applicationInput)
+	require.NoError(t, err)
+	defer director.UnregisterApplication(t, ORDContext.Context, ORDContext.DexGraphqlClient, ORDContext.Tenant, app.ID)
+
+	securedClient, configuration, certChain := getSecuredClientByContext(t, testCtx, runtime.ID)
```

Comment:
Probably reorder and move this secured client construction line below the register bundle + api (but before the test) so that the bundle + api are naturally after the application registration.